lib/plugins/aws/lib/monitorStack.js

```diff
@@ -74,7 +74,9 @@ module.exports = {
                     }
                     // Keep track of first failed event
                     if (eventStatus
-                      && eventStatus.endsWith('FAILED') && stackLatestError === null) {
+                      && (eventStatus.endsWith('FAILED')
+                      || eventStatus === 'UPDATE_ROLLBACK_IN_PROGRESS')
```

Comment:
@HyperBrain The main issue I noticed was that the error status from CloudFormation is only present in the 'UPDATE_ROLLBACK_IN_PROGRESS' event, not in the 'UPDATE_ROLLBACK_COMPLETE' event. I don't think it's a good solution on their side, but that's where it is today. If we don't capture the reason data from that event, then nothing usable exists to pass onto the user.